ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/ethernet_interfaces.py

```diff
@@ -144,6 +144,7 @@ def _get_ethernet_interface_cfg(self, adapter: dict, node_index: int, connected_
             ),
             "speed": adapter.get("speed"),
             "shutdown": not adapter.get("enabled", True),
+            "validate_state": None if adapter.get("validate_state", True) else False,
```

Comment:
Why does it need to be `None` rather than simply `True`?

if `True` is ok you could achieve the same with

```suggestion
            "validate_state": adapter.get("validate_state", True),
```

Reply:
I don't think the key should be present in the structured_config if it's true, only if it's false. This could add a lot of unnecessary lines.